qml/StatisticsChartBox.qml

```diff
@@ -201,6 +209,28 @@ Rectangle {
                     onTriggered: {
                         running ? statisticsChartViewLoader.item.dynamicPause() : statisticsChartViewLoader.item.dynamicContinue()
                     }
+
+                }
+            }
+        }
+
+        QtDialogs.FileDialog {
+            id: csvDialog
+            title: "Choose a file name and path to save csv"
+            folder: shortcuts.home
+            selectMultiple: false
+            selectExisting: false
+            selectFolder: false
+            defaultSuffix: ".csv"
```

Comment:
```suggestion
            defaultSuffix: ".csv"
            nameFilters: [ "CSV files (*.csv)", "All files (*)" ]
```

Reply:
This does not force the file name, but just shows in folder other files of same type

Reply:
I know but imagine that you want to replace another file. Selecting the option to see just the .csv files is easier to locate the file you are looking for. Also it's useful to see what .csv files you've already saved.